spec/03-plugins/31-proxy-cache/03-api_spec.lua

```diff
@@ -259,6 +259,17 @@ describe("Plugin: proxy-cache", function()
         assert.res_status(200, res)
         assert.same("Miss", res.headers["X-Cache-Status"])
       end)
+      it("miss if pass the Accept-Encoding", function()
+        local res = assert(proxy_client:get("/get", {
+          headers = {
+            host = "route-1.test",
+            ["kong-debug"] = 1,
+            ["Accept-Encoding"] = "gzip",
+          }
+        }))
+        assert.res_status(200, res)
+        assert.same("Miss", res.headers["X-Cache-Status"])
+      end)
```

Comment:
Maybe we could send another request to ensure then the gziped body is cached

Reply:
@StarlightIbuki Sure, thanks for your review. I have added this in last commit, plz take a look again.